Announce pending shutdown on stdout

tbg opened the pull request:
Before:

```
21:35 $ ./cockroach start --insecure
build:     alpha.v1-681-ge48daa5 @ 2016/03/06 02:34:04 (go1.6)
admin:     http://macts.local:26257
sql:       postgresql://root@macts.local:26257?sslmode=disable
logs:      cockroach-data/logs
store[0]:  path=cockroach-data
^C
```

After:

```
$ ./cockroach start --insecure
build:     alpha.v1-681-ge48daa5-dirty @ 2016/03/06 02:40:02 (go1.6)
admin:     http://macts.local:26257
sql:       postgresql://root@macts.local:26257?sslmode=disable
logs:      cockroach-data/logs
store[0]:  path=cockroach-data
^Cinitiating graceful shutdown of server
server drained and shutdown completed
```

<!-- Reviewable:start -->

[<img src="https://reviewable.io/review_button.svg" height="40" alt="Review on Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/4898)

<!-- Reviewable:end -->

tamird commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[cli/start.go, line 204 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4898#-KC8ntU7EgEFHbhQwCwC:-KC8ntU7EgEFHbhQwCwD:-789162242) ([raw file](https://github.com/cockroachdb/cockroach/blob/ae6cd520a1061f51b1c3432b2e029e75a6fba837/cli/start.go#L204)):*
const above but not here?

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4898#-:-KC8nwsdZ7inLVsHiN7J:1417088721)*
<!-- Sent from Reviewable.io -->

tbg committed:
Announce pending shutdown on stdout

Before:
```
21:35 $ ./cockroach start --insecure
build:     alpha.v1-681-ge48daa5 @ 2016/03/06 02:34:04 (go1.6)
admin:     http://macts.local:26257
sql:       postgresql://root@macts.local:26257?sslmode=disable
logs:      cockroach-data/logs
store[0]:  path=cockroach-data
^C
```

After:
```
$ ./cockroach start --insecure
build:     alpha.v1-681-ge48daa5-dirty @ 2016/03/06 02:40:02 (go1.6)
admin:     http://macts.local:26257
sql:       postgresql://root@macts.local:26257?sslmode=disable
logs:      cockroach-data/logs
store[0]:  path=cockroach-data
^Cinitiating graceful shutdown of server
server drained and shutdown completed
```

tschottdorf commented:
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[cli/start.go, line 204 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/cockroach/4898#-KC8ntU7EgEFHbhQwCwC:-KC8odyoUFE2_5FuF9CM:-499146009) ([raw file](https://github.com/cockroachdb/cockroach/blob/ae6cd520a1061f51b1c3432b2e029e75a6fba837/cli/start.go#L204)):*
Done.

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/cockroachdb/cockroach/4898)*
<!-- Sent from Reviewable.io -->
